### PR TITLE
Minor updates to build process

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,8 +25,6 @@
     "lib": "lib/"
   },
   "files": [
-    "LICENSE",
```
Comment (taion) [inline package.json]: These don't need to be explicitly specified.

File: package.json [hunk 2]
```diff
@@ -89,7 +87,6 @@
     "less-loader": "^2.2.1",
     "marked": "^0.3.5",
     "mocha": "^2.3.3",
-    "mt-changelog": "^0.6.2",
```
Comment (taion) [inline package.json]: Per https://github.com/AlexKVal/release-script/blob/05ab23673b040549bdece1802a262f3744c4beb6/src/release.js#L27-L32, this is enough to disable automated changelog generation.

**Comment by taion** [comment]
Note: until https://github.com/npm/npm/pull/11995 lands, we'll need to use npm@2 to publish releases.

**Comment by mtscout6** [comment]
Should that be added to the Contributing guide then for posterity? It'll be hard to find if it's hiding in this PR comment.

**Comment by taion** [comment]
I'll update `MAINTAINING.md`.

**Comment by taion** [comment]
Actually, I take that back, it looks like that PR will land any minute now, so the caveat won't be necessary. I'll still update the maintaining guide regarding changelog procedures though.

**Comment by taion** [comment]
Updated.

**Comment by taion** [comment]
I want to cut v0.28.4 soon. Are we okay with this?

**Pull request merged**